ample/modelling/rosetta_model.py

```diff
@@ -196,9 +202,9 @@ def ab_initio_cmd(self):
                 cmd += ["-psipred_ss2", self.psipred_ss2]
 
         if self.all_atom:
-            cmd += ['-return_full_atom true']
+            cmd += ['-return_full_atom', 'true']
         else:
-            cmd += ['-return_full_atom false']
+            cmd += ['-return_full_atom', 'false']
```

Comment:
lowercase here, in lines 243/244 you use uppercase

Reply:
good spot!